webpack/webpack.config.js

```diff
@@ -19,7 +19,6 @@ const when = (condition, config, negativeConfig) =>
 // primary config:
 const outDir = path.resolve(__dirname, project.platform.output);
 const srcDir = path.resolve(__dirname, 'src');
-const nodeModulesDir = path.resolve(__dirname, 'node_modules');
```

Comment:
This one is used by babel, that's why it failed on 3 OSes.
Need to wrap the line in `@if babel`

Reply:
I'm not sure how to squash those 2 commits into 1.
can you help me?